### ISSUE TITLE
Widget <fieldset> missing descriptive <legend>

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#13]
<fieldset> for widget settings missing descriptive <legend>
==>Add <legend> to label the fieldset, saying that it is for widget settings.
